### ISSUE TITLE
Catalyst Colors -- Easy toggles in settings to switch colorscheme 

### ISSUE THREAD

**Issue by jdev082** [CatalystDevOrg/Catalyst#104]
- Default
- Red
- Blue
- Green
- Orange
and light and dark varients

**Comment by webdev03** [comment]
I can try to figure out how to make a custom selector for this with Tailwind...

**Comment by webdev03** [comment]
Hmm... we can't do this with tailwind..

**Comment by JaydenDev** [comment]
@webdev03 do it how you did dark mode
that dark: prefix for dark mode

**Comment by webdev03** [comment]
I don't know how to make new modifiers.

**Comment by webdev03** [comment]
Instead, userstyles in the future.

**Comment by JaydenDev** [comment]
@webdev03 you MADE dark mode, wdym.

**Comment by JaydenDev** [comment]
It's near impossible for an average user to just code a stylesheet
@webdev03 
It needs to be like other browsers with pre-made customization options.
Right now it has barely any customization.

**Comment by webdev03** [comment]
Tailwind includes `dark:` but Tailwind or Windi (tailwind alternative) doesn't have support for custom ones like `red:`

**Comment by webdev03** [comment]
We can inject our OWN stylesheets, though.

**Comment by JaydenDev** [comment]
@webdev03 Well make color options then, preferably a drop down or some other selection thingy

**Comment by webdev03** [comment]
I'll work on this later.

**Comment by JaydenDev** [comment]
It's a feature, gonna wait on it for 2.0
For now, point releases for 1.0, bug fixes.

**Comment by webdev03** [comment]
Can I rename `On Hold` to `on hold`?

**Comment by JaydenDev** [comment]
@webdev03 why? sure?

**Comment by webdev03** [comment]
Done.

**Comment by JaydenDev** [comment]
Impossible to implement, closing.

**Comment by webdev03** [comment]
It's not impossible. Inject stylesheets into the window.

**Comment by JaydenDev** [comment]
@webdev03 If I'm doing it then it is

**Comment by webdev03** [comment]
You should do something in this repository.. you laid out the base, but all of the new things have got too complicated...

**Comment by hello-smile6** [comment]
> You should do something in this repository.. you laid out the base, but all of the new things have got too complicated...

I agree.

**Comment by JaydenDev** [comment]
@webdev03 The most I can do is front-end, (HTML and CSS, and very little JS)

**Comment by hello-smile6** [comment]
> @webdev03 The most I can do is front-end, (HTML and CSS, and very little JS)

Then why not work on something you have more experience with?

**Comment by JaydenDev** [comment]
@hello-smile6 the code is too flippin complicated, the only one that knows how to work it is @webdev03

**Comment by JaydenDev** [comment]
dont comment on closed issues

**Comment by jdev082** [comment]
Working on it!

**Comment by jdev082** [comment]
@webdev03 As I am working towards converting to standard CSS from Tailwind on the develop branch, that isn't relevant anymore.

**Comment by imdevarsh** [comment]
Okay. I think that if you use [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) this will be simple.

**Comment by jdev082** [comment]
> Okay. I think that if you use [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) this will be simple.

There's still bugs to fix before implementing this that I haven't been able to figure out.

**Comment by jdev082** [comment]
superseded by the implementation of custom CSS in v3.5.0-rc.1

**Comment by jdev082** [comment]
Will be implemented in v4.0.0

**Comment by jdev082** [comment]
This has been implemented
